packages/apps-config/src/endpoints/testingRococo.ts

```diff
@@ -294,6 +294,14 @@ export function createRococo (t: TFunction): LinkOption[] {
         providers: {
           'PHOENIX Protocol': 'wss://phoenix-ws.coinid.pro'
         }
+      },
+      {
+        info: 'rococoSunrock',
+        paraId: 499,
+        text: t('rpc.rococo.sunrock', 'Sunrock', { ns: 'apps-config' }),
+        providers: {
+          Sunrock: 'ws://sunrock.kaki.dev:9944'
```

Comment:
The issue with `ws` is that is not usable on the https:// site - you cannot make connections to non-secure origins from from HTTPS unless it is localhost.

Reply:
@jacogr can I simply remove the provider ? 
This is an ephemeral network (but Shawn asked me to still add info in the UI) and I don't want to add SSL support

Reply:
I reset it to wss, and will take care of the ssl part later today